src/bit-systems/media-loading.ts

```diff
@@ -19,6 +20,39 @@ import { loadAudio } from "../utils/load-audio";
 import { MediaType, mediaTypeName, resolveMediaInfo } from "../utils/media-utils";
 import { EntityID } from "../utils/networking-types";
 
+export const MediaContentBounds = (MediaLoaded as any).contentBounds as Map<EntityID, Vector3>;
+
+const getBox = (() => {
+  const rotation = new THREE.Euler();
+  return (world: HubsWorld, eid: EntityID, rootEid: EntityID, worldSpace?: boolean) => {
+    const box = new THREE.Box3();
```

Comment:
You may import `Euler` and `Box3`, and remove `THREE.` prefix here.

Reply:
Opps, yep, updated.